frontend/packages/noobaa-storage-plugin/integration-tests/views/createBC.view.ts

```diff
@@ -48,11 +46,26 @@ const getBackingStoreRequiredCount = (tiers: Tier[]) => {
 };
 
 const createDummyBackingStore = (name: string) => {
-  const defaultBS = JSON.parse(
+  let defaultBS = null;
+  try {
+    defaultBS = JSON.parse(
+      execSync(
+        'kubectl get backingstores.noobaa.io noobaa-default-backing-store -n openshift-storage -o json',
+      ).toString(),
+    );
+  } catch {
+    // Create a default backing store by ourselves
+    // Create secret to be consumed by the backingstore
     execSync(
-      'kubectl get backingstores.noobaa.io noobaa-default-backing-store -n openshift-storage -o json',
-    ).toString(),
-  );
+      `oc create secret generic test-secret-for-bs -n openshift-storage --from-literal=password='iamthewatcherofthewall'`,
+    );
+    execSync(`echo '${JSON.stringify(mockDefaultBS)}' | kubectl create -n openshift-storage -f -`);
+    defaultBS = JSON.parse(
+      execSync(
+        'kubectl get backingstores.noobaa.io noobaa-default-backing-store -n openshift-storage -o json',
+      ).toString(),
+    );
+  }
```

Comment:
On OCS , default BS is required to be present after installation.
Why are we mocking it up in case of absence ?

Reply:
Because if Noobaa creation is stuck(happens in OCP) we need to manually create one.

Reply:
That we might need to tackle separately. From OCS point of view a default backing store should be created. And if we mock it up , then we might miss a valid check.

Reply:
yes, there are separate tests that are written as part of the installation flow that would help us deduce that default BC has indeed been created. I guess you are not following our testing efforts and test cases.